tests/common/ixia/ixia_helpers.py

```diff
@@ -0,0 +1,439 @@
+# -*- coding: utf-8 -*-
+"""This module contains the high-level wrapper function using the APIs defined
+by Ixia/Keysights ixnetwork_restpy library functions. Intention of providing
+these to SONiC group is to avoid writing multiple low-level rest API calls for
+doing the top-level tasks like configure ports, create topology, 
+start protocols, start traffic etc.
+
+This module also contains a definition of a simple helper class 
+"IxiaFanoutManager" which can be used to manage cards and ports of ixia 
+chassis instead of reading it from fanout_graph_facts fixture.
+"""
+
+import re 
+from common.reboot import logger
+from ixnetwork_restpy import SessionAssistant, Files
+import pytest
+
+class IxiaFanoutManager () :
+    """Class for managing multiple chassis and extracting the information 
+     like chassis IP, card, port etc. from fanout_graph_fact."""
+
+    def __init__(self,fanout_data) :
+        """ When multiple chassis are available inside fanout_graph_facts 
+        this method makes a  list of chassis connection-details out of it.
+        So each chassis and details  associated with it can be accessed by 
+        a integer index (starting from 0)
+
+        Args:
+           fanout_data (dict): the dictionary returned by fanout_graph_fact
```

Comment:
Can you give a simple example of fanout_data here?

Reply:
I have provided an example of fan-out data in the comment section.